### PR TITLE
Clean up the terminology related to build environment

### PR DESCRIPTION
**Pull request by martyone**

### CODE REVIEW

File: Develop/Collaborate/Contribution_Example/README.md [hunk 1]
```diff
@@ -59,10 +59,10 @@ Installed Size : 6.4 MiB
 Summary        : Headers for xulrunner
 Description    : abzaza
   Development files for xulrunner.
+```
 
-# she now knows that it comes from the "jolla" repository.
-# she goes back to her development terminal, and
-# enters the build target to update the ssu repositories.
+9\) She now knows that it comes from the "jolla" repository. She goes back to her development terminal, and enables the repository under her build environment the way suggested in (Enabling Additional Package Repositories)[/Tools/Sailfish_SDK/Building_packages/README.md#enabling-additional-package-repositories]:
```
Comment (vigejolla) [inline Develop/Collaborate/Contribution_Example/README.md:64]:
```suggestion
9\) She now knows that it comes from the "jolla" repository. She goes back to her development terminal, and enables the repository under her build environment the way suggested in (Enabling Additional Package Repositories)[/Tools/Sailfish_SDK/Building_packages/#enabling-additional-package-repositories]:
```

File: Tools/Sailfish_SDK/Building_packages/README.md [hunk 2]
```diff
@@ -102,9 +110,9 @@ It should be noted that device or hardware-platform-specific packages (especiall
 
 ## Building Binaries Locally
 
-In some cases, you will not want to build an entire package (.rpm) to install, but instead want to build a single binary from a simple (most likely Qt-based) project. This can be handy, for example, during testing or prototyping.
+In some cases, you will not want to build an entire package (.rpm) to install, but instead want to build a single binary from a simple (most likely Qt-based) project for which you do not have an RPM SPEC file yet. This can be handy, for example, during testing or prototyping.
 
-In this case, you simply need to build the project from within a build-shell. For example, to build a simple Qt-based project (called "test", located under `~/test/` of the host) the following steps could be taken:
+In this case, you simply need to invoke the build commands directly from within the build environment. For example, to build a simple Qt-based project (called "test", located under `~/test/` of the host) the following steps could be taken:
```
Comment (vigejolla) [inline Tools/Sailfish_SDK/Building_packages/README.md:115]: The name of the project is not relevant here, so I would omit the 'called "test"' part.

File: Develop/Collaborate/Contribution_Example/README.md (hunk 1 above)
Comment (martyone) [reply Develop/Collaborate/Contribution_Example/README.md]: Fixed and fixed the syntax too.

File: Tools/Sailfish_SDK/Building_packages/README.md (hunk 2 above)
Comment (martyone) [reply Tools/Sailfish_SDK/Building_packages/README.md]: Dropped

**Review by vigejolla — APPROVED** [review]
LGTM

**Comment by martyone** [comment]
Rebased

**Pull request merged**